frontend/src/metabase/query_builder/components/BreakoutPopover.jsx

```diff
@@ -26,6 +26,8 @@ const BreakoutPopover = ({
       className={className}
       width={width}
       field={breakout}
+      query={query}
+      metadata={query.metadata()}
```

Comment:
Passing these components to `FieldList` because this "ui" component instantiates Dimensions, so we need access to queries to properly instantiate the dimensions of models, nested questions, etc.